src/main/java/com/example/bookstoreapp/models/CatalogItem.java

```diff
@@ -14,16 +14,19 @@ public class CatalogItem {
   private String name;
   private Float price;
   private Integer stockQuantity;
+  private Integer itemLimit;
 
   public CatalogItemEntity toEntity() {
-    return new CatalogItemEntity(this.getId(), this.getName(), this.getPrice(), this.getStockQuantity());
+    Integer itemLimits = (this.getItemLimit() == null) ? 0 : this.getItemLimit();
```

Comment:
itemLimits represent multiple itemLimit.
It should not be plural name.

Reply:
fixed the variable name